content/docs/join_us/recruitment_FAQs.md

```diff
@@ -19,7 +19,7 @@ Questions are grouped by:
 
 - [Application Stage](#application-stage)
 - [Profile](#profile)
-- [Position](#position)
+- [Role](#role)
 - [Interview](#interview)
 - [Offer](#offer)
 
```

Comment:
It occurs to me that maybe applicants should get in touch with recruitment rather than HR for advice on visas (point below)? I can't seem to find any info on it though - any ideas @penelopeysm